Tasks/KubernetesV1/src/utilities.ts

```diff
@@ -73,4 +73,11 @@ export function assertFileExists(path: string) {
         tl.error(tl.loc('FileNotFoundException', path));
         throw new Error(tl.loc('FileNotFoundException', path));
     }
+}
+
+export function writeInlineConfigInTempPath(inlineConfig: string): string {
+    var tempInlinePath = getNewUserDirPath();
+    tempInlinePath = path.join(tempInlinePath, "inlineconfig.yaml");
```

Comment:
if there are 2 tasks both with inline files, does the second one overwrite the first file?

Reply:
Each call will create a new dir path based on timestamp. So there will be no overwrite of files. And as soon as a task is complete(succeeded/failed), this file is cleaned up